keps/sig-auth/3257-clustertrustbundles/README.md

```diff
@@ -548,7 +582,7 @@ https://storage.googleapis.com/k8s-triage/index.html
 The following scenarios will need to be tested using integration tests:
```

Comment:
The bulk of these will need to be e2es because cannot run the kubelet in integration tests.

Reply:
I think they're actually all already called out as e2e tests.  I've removed them from the integration test section.